### ISSUE TITLE
RFC: Semantics in qr_flutter

### ISSUE THREAD

**Issue by nilsreichardt** [theyakka/qr.flutter#132]
## Description
As I descripted in #101 the `QrPainter` Widget has no built in accessibility support for screen readers. So it's not possible to pass a `semanticLabel`, like `QrImage` provides it.

@lukepighetti wrote in #101:
> We're going to recommend that you tag this with your own Semantics. Closing this PR because there is nothing actionable for us.

This an option, but if we decide to choose this option this should better documented (e.g. a headline "Accessibility" in the README.md that `QrPainter` has no built in accessibility support and with the hint to use your own `Semantics` to support screen reader).

**Comment by lukepighetti** [comment]
It's not clear to me how we should tag this. Wouldn't the semantics be based on the implementation? If you put a QRCode on a button, wouldn't you want to use button semantics? If a QRCode is a picture, wouldn't you use some type of image context? A little more description from your end on what prompted creating this issue would be super helpful. It doesn't seem like something we should document either, as it seems like it would shake out on the consuming side. But I'm happy to hear more and possibly learn something new.

**Comment by nilsreichardt** [comment]
I have already implemented screen reader support for `QrImage`: https://github.com/theyakka/qr.flutter/pull/93.

You can use it like:
```dart
QrImage(
  data: "23CAS",
  semanticsLabel: "Code for joining the group as qr code"
  size: 200.0,
),
```

Similar behavior has `flutter_svg` or a `Image`:
```dart
SvgPicture.asset(
  assetName,
  color: Colors.red,
  semanticsLabel: 'A red up arrow'
);
```

But the `QrPainter` Widget has no screen reader support. I would expect to have also a `semanticsLabel` for `QrPainter`, like:
```dart
QrPainter.withQr(
  qr: _qr!,
  semanticsLabel: "describing the qr code"
);
```

Otherwise screen reader can not detected the qr code. You can see in this video. TalkBalk is not able to see the qr code (green border = focused element by the screen reader):
![QrPainter](https://user-images.githubusercontent.com/24459435/87852330-43ed5900-c901-11ea-92f3-5fc5ce5071c7.gif)

So my thoughts are: Either implementing a `semanticsLabel` parameter (like I did it with `QrImage`, but `QrPainter` is very low level, so this could be difficult) or add a note (e. g. in `README.md`) that `QrPainter` will not be detected by screen reader. To solve this, you can use the `Semantics` widget like:
```dart
return Container(
  width: 280,
  child: Semantics(
    label: 'describing the qr code',
    child: CustomPaint(
      size: Size.square(200),
      painter: QrPainter(
        data: message,
        version: QrVersions.auto,
      ),
    ),
  ),
);
```

**Comment by lukef** [comment]
I believe this is fixed and included? It's on the 5.0 branch at least.
